### PR TITLE
Create SETUP_GUIDELINES.md

### PR DESCRIPTION
**Pull request by PrajwalM2212**
## Description

Created setup guidelines to help new developers contribute to the project.

## Related issues and discussion
#865 
 
 ## Screen-shots, if any
setup guidelines

### CODE REVIEW

File: SETUP_GUIDELINES.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+## Steps for Developers 
+1. Download the latest stable release of [Android Studio](https://developer.android.com/studio/index.html).
+2. Fork the [Project Repository](https://github.com/openfoodfacts/openfoodfacts-androidapp) by clicking "Fork".
+3. Clone the repository `git clone https://github.com/YOUR_USERNAME/openfoodfacts-androidapp.git`.
+4. Initialize Git `git init`.
```
Comment (sachincool) [inline SETUP_GUIDELINES.md]: Cloning automatically creates the .git  repository inside the repo  you don't need to explicitly  create it. 
You can put here `cd openfoodfacts-androidapp`

File: SETUP_GUIDELINES.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+5. Run the project in Android Studio 
+6. Fetch the latest version of the code from branch "master".
```
Comment (sachincool) [inline SETUP_GUIDELINES.md]: If the user cloned the repo it automaticaly contains the latest commits. you dont need to fetch again and again. you can remove this one.

File: SETUP_GUIDELINES.md (hunk 1 above)
Comment (PrajwalM2212) [reply SETUP_GUIDELINES.md]: Yes . I will make the change . thanks @sachincool

Comment (PrajwalM2212) [reply SETUP_GUIDELINES.md]: @sachincool  Thanks ! I will make the required change .

File: SETUP_GUIDELINES.md (hunk 2 above)
Comment (PrajwalM2212) [reply SETUP_GUIDELINES.md]: Thanks !

**Comment by PrajwalM2212** [comment]
@sachincool  I will add the links.

File: SETUP_GUIDELINES.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+## Steps for Developers 
+1. Download the latest stable release of [Android Studio](https://developer.android.com/studio/index.html).
+2. Fork the [Project Repository](https://github.com/openfoodfacts/openfoodfacts-androidapp) by clicking "Fork".
+3. Clone the repository <br>`git clone https://github.com/YOUR_USERNAME/openfoodfacts-androidapp.git`.
+4. Change directory `cd openfoodfacts-androidapp`.
+5. Run the project in Android Studio
+6. Take a look at [README.md](https://github.com/openfoodfacts/openfoodfacts-androidapp/blob/master/README.md)
```
Comment (sachincool) [inline SETUP_GUIDELINES.md]: @PrajwalM2212  i didn't meant to put the Readme.md link here.  I meant to put this `Setup-guidelines.md` link inside Readme  So users can navigate.   you can put it in  #Installation. 
You can use this:-   `If you're a new contributor to the open-source, We recommend you to read our [_SETUP-GUIDELINES_](Link_Here)`

Comment (PrajwalM2212) [reply SETUP_GUIDELINES.md]: Ok , I get it . Thanks for the help @sachincool

**Comment by teolemon** [comment]
looks good to me, seconds opinions ?

**Comment by teolemon** [comment]
merged, thanks @PrajwalM2212 @sachincool @ross-holloway94

**Pull request merged**